.github/workflows/ansible.yml

```diff
@@ -22,7 +22,9 @@ jobs:
       IMAGE: 'fedora:34'
     strategy:
       matrix:
-        extra: ["method=pip", "method=copr", "method=official", "method=pip avocado_vt=true", "method=copr avocado_vt=true", "method=official avocado_vt=true"]
+        #        extra: ["method=pip", "method=copr", "method=official", "method=pip avocado_vt=true", "method=copr avocado_vt=true", "method=official avocado_vt=true"]
+        #        lack of updated packages in the repos for python3-avocado-plugins-vt
+                extra: ["method=pip", "method=copr", "method=official", "method=pip avocado_vt=true"]
```

Comment:
Oh, now I know what I'm missing.

About `avocado-vt`, it'd be nice to enhance the playbooks to test the copr builds when `method=copr`, and later enhance both avocado and avocado-vt when method is `official`, but with official meaning the Fedora/EPEL packages.

Reply:
oh great, then there are packages for avocado-vt. Could you please point me where are the Fedora/EPEL repos?

Reply:
> oh great, then there are packages for avocado-vt. Could you please point me where are the Fedora/EPEL repos?

Sure, all COPR repos are located at: https://copr.fedorainfracloud.org/groups/g/avocado/coprs/

And the Fedora/EPEL packages are on the "modular" repos, so you'd need to run:

```
dnf module -y enable avocado:latest
dnf module -y enable avocado-vt:latest
```